ticTacToe.js

```diff
@@ -1,6 +1,663 @@
+		winner = "player X";
+	} else if(gameBoard[0][2] === 1 && gameBoard[1][2] === 1 && gameBoard[2][2] === 1) {
+		winner = "player X";
+	} else if(gameBoard[0][0] === 1 && gameBoard[1][1] === 1 && gameBoard[2][2] === 1) {
+		winner = "player X";
+	} else if(gameBoard[0][2] === 1 && gameBoard[1][1] === 1 && gameBoard[2][0] === 1) {
+		winner = "player X";
+	} else if(gameBoard[0][0] === 2 && gameBoard[0][1] === 2 && gameBoard[0][2] === 2) {
+		winner = "player O";
+	} else if(gameBoard[1][0] === 2 && gameBoard[1][1] === 2 && gameBoard[1][2] === 2) {
+		winner = "player O";
+	} else if(gameBoard[2][0] === 2 && gameBoard[2][1] === 2 && gameBoard[2][2] === 2) {
+		winner = "player O";
+	} else if(gameBoard[0][0] === 2 && gameBoard[1][0] === 2 && gameBoard[2][0] === 2) {
+		winner = "player O";
+	} else if(gameBoard[0][1] === 2 && gameBoard[1][1] === 2 && gameBoard[2][1] === 2) {
+		winner = "player O";
+	} else if(gameBoard[0][2] === 2 && gameBoard[1][2] === 2 && gameBoard[2][2] === 2) {
+		winner = "player O";
+	} else if(gameBoard[0][0] === 2 && gameBoard[1][1] === 2 && gameBoard[2][2] === 2) {
+		winner = "player O";
+	} else if(gameBoard[0][2] === 2 && gameBoard[1][1] === 2 && gameBoard[2][0] === 2) {
+		winner = "player O";
+	} 
+
+}
+
+	//set up all the squares with null
+	clearBoard();
+	alert("Player X, it's your turn first!");
+
+	//what to do when the top-right square is clicked
+	$('#one').click(function() {
+		//check to make sure the square is still null
+		if(gameBoard[0][0] === null) {
+			//check to see if it's player X's turn
+			if(turn === x) {
+				//place an x in the square
+				$(this).text('x');
+				//set the gameBoard array to var x
+				gameBoard[0][0] = x;
+				//change to player O's turn
+				turn = o;
+				//what to do if it was not player X's turn
+			} else {
+				//place an O in the square
+				$(this).text('o');
+				//set the gameBoard array to var o
+				gameBoard[0][0] = o;
+				//change to player X's turn
+				turn = x;
+			}
+			//if the square is not null let the player know
+		} else {
+			alert("That square is already taken");
+		}
+		// console.log(gameBoard);
+		//check to see if the game is over
+		isGameOver();
+		//alert the next player it's their turn
+		if(turn === x) {
+			alert("Player X, it's your turn next!");
+		} else {
+			alert("Player O, it's your turn next!");
+		}
+	});
+	
+	//repeat same as above for when each square is clicked
+	$('#two').click(function() {
+		if(gameBoard[0][1] === null) {
+				if(turn === x) {
+				$(this).text('x');
+				gameBoard[0][1] = x;
+				turn = o;
+			} else {
+				$(this).text('o');
+				gameBoard[0][1] = o;
+				turn = x;
+			}
+		} else {
+			alert("That square is already taken");
+		}
+		
+		isGameOver();
+		if(turn === x) {
+			alert("Player X, it's your turn next!");
+		} else {
+			alert("Player O, it's your turn next!");
+		}
+	});
+
+	$('#three').click(function() {
+		if(gameBoard[0][2] === null) {
```

Comment:
This logic is extremely similar for each of the squares.  Is there anyway you could make a function that accepts arguments about which square and attach the same function to each square?  Or maybe the function could determine which square it is when it's called using `this`?